File: src/main/java/com/pfe/elearning/profile/dto/Response/ProfileResponse.java
package com.pfe.elearning.profile.dto.Response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@Getter
@AllArgsConstructor
@Builder
public class ProfileResponse {

    private Long profileId;
    private String firstName;
    private String lastName;
    private String email;
    private String image; // Add image field
    private String description; // Add description field
    private String phoneNumber; // Add phoneNumber field
    private Date dateOfBirth; // Add dateOfBirth field
    private String country; // Add country field
    private String currentJob; // Add currentJob field
    private int experience; // Add experience field
    private String degreeOfEducation; // Add degreeOfEducation field
    private String certificates; // Add certificates field
}

File: src/app/models/profile/profile-response.ts
export interface ProfileResponse {
  profileId?: number;
  firstName?: string;
  lastName?: string;
  email?: string;
  image?: string;
  description?: string;
  phoneNumber?: string;
  dateOfBirth?: Date;
  country?: string;
  currentJob?: string;
  experience?: number;
  degreeOfEducation?: string;
  certificates?: string;
  // Add other fields as needed
}
